spec/system/candidate_interface/apply_again/candidate_applies_again_with_three_choices_spec.rb

```diff
@@ -0,0 +1,190 @@
+require 'rails_helper'
```

Comment:
Could just call this spec `candidate_applies_again_spec.rb` because 3 choices will be the norm once the FF is deleted.

Reply:
Good point. Fixed